pkg/apis/core/v1/helper/helpers_test.go

```diff
@@ -806,3 +808,100 @@ func TestLoadBalancerStatusEqual(t *testing.T) {
 		})
 	}
 }
+
+func TestIsExtendedResourceName(t *testing.T) {
+	type testCase struct {
+		name     v1.ResourceName
+		expected bool
+	}
+
+	testCases := []testCase{
+		{
+			name:     "pod.alpha.kubernetes.io/opaque-int-resource-foo",
+			expected: false,
+		},
+		{
+			name:     "a/b",
+			expected: true,
+		},
+		{
+			name:     "a/",
+			expected: false,
+		},
+		{
+			name:     v1.DefaultResourceRequestsPrefix + "name",
+			expected: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(string(tc.name), func(t *testing.T) {
+			result := IsExtendedResourceName(tc.name)
+			if result != tc.expected {
+				t.Errorf("Expected %v for resource name %s, got %v", tc.expected, tc.name, result)
+			}
+		})
+	}
+}
+
+func TestIsHugePageResourceName(t *testing.T) {
+	type args struct {
+		name v1.ResourceName
+	}
```

Comment:
Why define `args` struct here? It is not used in the test.

Reply:
done